test/status.spec.js

```diff
@@ -426,4 +427,39 @@ describe('Status metadata', () => {
       });
     });
   });
+
+  describe('Links', () => {
+    describe('hasNext', () => {
+      it('Returns true when next link exists', () => {
+        const status = updateStatus(
+          createStatus(),
+          {
+            links: {
+              next: 'http://link-to-next',
+            },
+          },
+        );
+        const obj = {};
+        setStatus(obj, status);
+
+        expect(hasNext(obj)).to.be.true;
+      });
+
+
+      it('Returns false when next link doesn\'t exist', () => {
+        const status = updateStatus(
```

Comment:
I would just add here a case when there is no `links` object at all in the status.